core/thread/fetch_spec.rb

```diff
@@ -0,0 +1,15 @@
+ruby_version_is '2.5' do
```

Comment:
Can you require the `spec_helper` here like in other files?

Reply:
Do we need it? I ran the specs locally it passed. Hence I didn't include it.

Reply:
It's notably useful to run the spec directly like `$ ruby core/thread/fetch_spec.rb`.
There is also some setup related to `Thread.report_on_exception` which should be done before any spec.

Reply:
done. I have updated the PR.